resources/language/resource.language.sv_se/strings.po

```diff
@@ -239,38 +239,52 @@ msgid "Appearance"
 "* That PIN is then used to generate a much stronger and complexer encryption key using Scrypt.\n"
-"* As a user you will not need to know this complex key.\n\n"
+"* As a user you will not need to know this complex key.\n"
+"\n"
 "[COLOR gold][B]Storing sensitive data[/B][/COLOR]\n"
-"Whenever sensitive data is being [B]stored[/B]:\n\n"
+"Whenever sensitive data is being [B]stored[/B]:\n"
+"\n"
 "* The user will be asked for their PIN first.\n"
-"* After a correct PIN the actual data to store will be asked.\n\n"
+"* After a correct PIN the actual data to store will be asked.\n"
+"\n"
 "[COLOR gold][B]Retrieving sensitive data[/B][/COLOR]\n"
-"Whenever sensitive data is being [B]retrieved[/B]:\n\n"
+"Whenever sensitive data is being [B]retrieved[/B]:\n"
+"\n"
 "* The user will only be asked for their PIN.\n"
-"* That PIN is then used to securely fetch the actual data.\n\n"
+"* That PIN is then used to securely fetch the actual data.\n"
+"\n"
 "[COLOR gold][B]Resetting the Vault[/B][/COLOR]\n"
 "If a user has forgotten the PIN, there is no way of recovering the data. The only solution is to reset the Vault and pick a new PIN and generate a new Vault Key. [B]This will invalidate all encrypted data[/B]. This data must be entered again."
-msgstr "[COLOR gold][B]Retrospect lösenordsvalv[/B][/COLOR]\n\n"
-"'Lösenordsvalvet' gör det möjligt att spara känslig data på ett säkrare sätt. Så här fungerar det:\n\n"
-"* Användaren väljer en enkel PIN-kod för valvet som används som en huvudkod för att komma åt innehållet i valvet.\n"
-"* PIN-koden kan vara kort och enkel att komma ihåg: till exempel 3145.\n"
-"* PIN-koden används sedan till att generera en mycket starkare och komplexare krypteringsnyckel med hjälp utav Scrypt.\n"
-"* Som användare behöver du inte känna till den här komplexa nyckeln.\n\n"
-"[COLOR gold][B]Lagra känslig data[/B][/COLOR]\n"
-"Varje gång som känslig data [B]lagras[/B]:\n\n"
-"* Användaren kommer först att bli ombedd att ange sin PIN-kod.\n"
-"* Efter en korrekt PIN-kod kommer de faktiska uppgifterna att lagras.\n\n"
-"[COLOR gold][B]Hämta känslig data[/B][/COLOR]\n"
-"Varje gång som känslig data [B]hämtas[/B]:\n\n"
-"* Användaren kommer då att bara bli ombedd att ange sin PIN-kod.\n"
-"* PIN-koden används sedan för att säkert hämta de faktiska uppgifterna.\n\n"
-"[COLOR gold][B]Återställa lösenordsvalvet[/B][/COLOR]\n"
-"Om en användare har glömt PIN-koden, finns det inget sätt att få tillbaka uppgifterna. Det enda sättet är att återställa valvet och att välja en ny PIN-kod och generera en ny valvnyckel. [B]Detta gör att det krypterade innehållet blir ogiltigt[/B]. Därför måste alla inloggningsuppgifter anges igen."
+msgstr ""
+"[COLOR gold][B]Retrospect lösenordsvalv[/B][/COLOR]\n"
+"'Valvet' gör det möjligt att spara känslig information på ett säkrare sätt. Så här fungerar det:\n"
+"\n"
+"* Användaren väljer en enkel kod till valvet som blir huvudkoden för att komma åt informationen i valvet.\n"
```

Comment:
Why did you remove the PIN part?

Reply:
Even in Kodi they use `Allmänt`

![image](https://user-images.githubusercontent.com/5789283/99158528-e2cd6a80-26d3-11eb-86c4-b7a66390d3dc.png)